src/_packagedcode/pypi.py

```diff
@@ -452,8 +453,9 @@ def parse_metadata(location, datasource_id, package_type):
         type=package_type,
         primary_language='Python',
         name=name,
-        version=version,
-        description=get_description(meta, location),
+        version=version, #TODO: https://github.com/nexB/scancode-toolkit/issues/3014
+        # description=get_description(meta, location),
```

Comment:
I am not sure that skipping the description is the right way to fix this... though this is less important here.